### ISSUE TITLE
Space Pin ASA restored to incorrect position on Hololens2(UWP) build

### ISSUE THREAD

**Issue by r01hee** [microsoft/MixedReality-WorldLockingTools-Unity#240]
Hello,

In late Aug. 2021,  Space Pin ASA restored to correct position.
The states at the time are following:
- Unity 2020.3.13f1
- MRTK 2.7.2
- Azure Spatial Anchors SDK Core 2.10.2
- Azure Spatial Anchors SDK for Windows 2.10.2
- OpenXR Plugin 1.2.3
- Mixed Reality OpenXR Plugin 1.0.2
- WorldLockingTools 1.4.2
-  Frozen World Engine 1.1.1
I'm not sure about HoloLens OS version and Visual Studio version at that time.

Then I edited PinTest Scene in WorldLocking.ASA.Examples
- MRTK Profile to "WLT HL2 MRTKProfile"
- Anchor Subsystem to "XRSDK" at "WorldLockingManager"
- "Coarse Relocation" to disabled at "Publisher ASA"

Then I built as `Release` / `ARM64` and ran on Hololens2.
1. I located Space Pins and pressed "Publish".
2. I restarted the application and pressed "Load File".
3. Then Scape Pins are restored at the expected place.

But, Recently (late Nov. 2021) I rebuilt this project and ran on same Hololens2, then Space Pins are restored at the unexpected place.
Note that I changed the project about followings.
- I recreated new Spatial Anchors resource and got new access key via Azure portal, and then applied the access key to the project.
- Versions of HoloLens OS and Visual Studio might be changed; currently, HoloLens: Windows 10 version20348.1438, Visual Studio: 2019 Version 16.9.1.

---

After that, I updated WorldLockingTools to v1.5.4, but it was no good.
Then I updated OpenXR Plugin to 1.2.0, OpenXR Plugin to 1.3.0, also no good.

---

And then, I [updated WorldLocking.ASA to use ASA SDK v2.11.0](https://github.com/r01hee/MixedReality-WorldLockingTools-Unity/commit/5c837b3a5e699f9585e7a698b6d3d0ab0408d935), it worked exactly as I expected.

Then I tried [ASA samples](https://github.com/Azure/azure-spatial-anchors-samples) (only ASA, not included WLT).
Even In ASA samples, It's good on ASA SDK 1.11.2, but it's not good on ASA SDK 1.10.2, in my environment.

So I hope that WorldLocking.ASA supports ASA SDK v1.11.2.

**Comment by fast-slow-still** [comment]
Thank you for the detailed description. This is very helpful!

**Comment by fast-slow-still** [comment]
I don't doubt the accuracy of this report (many thanks again for the details!!!), but I have not been able to reproduce the issue locally. Perhaps it is a HoloLens2 OS version issue? Or perhaps a MROpenXR Plugin issue that has been fixed (I tested with v1.2.0)?

Regardless, support for ASA v2.11.0 is in and has passed all tests (using MROpenXR Plugin 1.2.0).

PR #241 

Closing this issue for now. Thanks!
